tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -570,7 +570,7 @@ public:
    RFieldBase(RFieldBase&&) = default;
    RFieldBase& operator =(const RFieldBase&) = delete;
    RFieldBase& operator =(RFieldBase&&) = default;
-   virtual ~RFieldBase();
+   virtual ~RFieldBase() = default;
```

Comment:
This is the only change I see related to the error message pointing to `~RField() override = default;`